pages/index.js

```diff
@@ -60,10 +63,30 @@ const curUser = {
   country: "Germany",
   iban: "DE43500105175854591561",
 };
+
+const curArticles = [
+  {
+    articleNumber: 1,
+    articleName: "Kartoffeln",
+    price: 95,
+  },
+  {
+    articleNumber: 2,
+    articleName: "Eis",
+    price: 5,
+  },
+  {
+    articleNumber: 3,
+    articleName: "Pilze",
+    price: 10,
+  },
+];
+
 export default function Home() {
   const [client, setClient] = useState(curUser);
   const [nav, setNav] = useState("userCard");
   const [customers, setCustomers] = useState(curCustomers);
+  const [articles, setArticles] = useState(curArticles);
```

Comment:
I would highly recomment to write clean an the whole word `currentArticles`, `currentUser`, for the future You and external people whou would like to read your code :)

Reply:
Yes, i will keep it in mind.